sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
When you say "MAC protocol", what do you mean exactly by this? CSMA/CA is exactly the process of sensing the channel and depending on the outcome (and maybe a random backoff) the decision to send or not.

As far as I understand what you mean by "AUTOCCA", this seems equal.